tests/func/parsing/test_interpolated_entry.py

```diff
@@ -272,14 +273,17 @@ def test_cmd_dict(tmp_dir, dvc, bool_config, list_config):
         if list_config:
             conf["parsing"]["list"] = list_config
 
+    string = "spaced string"
+    mixed_quote_string = "quote\"'d"
```

Comment:
Seems like we should probably also be testing additional cases that contain backslashes on windows to make sure that we aren't mangling dicts that map to file paths

(see the rules for handling backslashes: https://github.com/python/cpython/blob/b399115ef18945f7526492225d72a512048ad20d/Lib/subprocess.py#L576)

Reply:
@pmrowla Sorry for the late reply, not sure I follow what additional cases we should test. 
Since we call directly `subprocess.list2cmdline` (windows) / `shlex.quote` , are we testing how those functions work?